### ISSUE TITLE
TAPE clears current file on Stop

### ISSUE THREAD

**Issue by okyeron** [monome/norns#766]
Just discovering TAPE today and I noticed the following:

- load a file and it cues the file to it's start point
- as soon as I hit "stop" the file is cleared and I need to re-select the file from the menu again

I would find it very useful if you could start/stop/start/stop/etc. the same file without having to reload and jump back to the start each time - oh wow... I just realized I'm asking for PAUSE instead of STOP

Use case: Load a long spoken word sample, play a section, stop (or PAUSE), then play another section, Pause, etc.

Even better would be the capability to scrub thru a Tape file by time index with the encoders.

**Comment by giovannilami** [comment]
I don’t know if to not make a loop is a precise choice, but it's strange since the old Norns' firmware looped tape' samples as default. It would be great to have this again in 2.0

**Comment by catfact** [comment]
no, it's not a deliberate choice, it's from the fact that new tape is made from scratch instead of using supercollider. i'll add the loop feature - it's easy but not trivial. 

(you're right to point out this was old default, since i'd forgotten - thanks!)

**Comment by okyeron** [comment]
@catfact is pause a doable thing as well? 

I like the idea that whatever might be loaded in Tape could stay there "paused" until a new file is loaded.

**Comment by catfact** [comment]
yes, i'll presently add loop (by default), pause (as an option? i'm not sure what the consensus is), and possibly seek (depending on how hard that proves. i think it will be fine.)

**Comment by giovannilami** [comment]
@catfact thanks a lot! ^^
sure, adding pause as an option would be something extremely useful.
ciao!

**Comment by catfact** [comment]
sorry about this one, has just been slipping through the cracks.

i started an ambitious rewrite of tape system with these and other features, but it has been languishing for a while.
